homeassistant/components/electric_kiwi/__init__.py

```diff
@@ -70,3 +73,34 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         hass.data[DOMAIN].pop(entry.entry_id)
 
     return unload_ok
+
+
+async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Migrate old entry."""
+    if config_entry.version == 1 and config_entry.minor_version == 1:
+        if config_entry.unique_id is not None and config_entry.unique_id.startswith(
+            DOMAIN
+        ):
```

Comment:
why would this not be the case?

Reply:
removed, your right it will always be the case. FYI I'm updating the PR for the new release of the API, it fixes a few of the weird items with the unique id so now its just the customer number which is part of the session object, instead of out of an array. Otherwise not much changes. Updating tests as well.